docs/_appendix/github-intro.md

```diff
@@ -170,6 +170,26 @@ them to be reviewed and merged.
 
 7.  Repeat these steps as many times as you'd like before you submit.
 
+## Verify
+
+The Web Platform Test project has an automated tool
+to verify that coding conventions have been verified,
```

Comment:
+to verify that coding conventions have been verified,

“to verify that coding conventions have been _followed_” might be better here (to avoid _verify_/_verified_ repetition)